service/tbc/websocket.go

```diff
@@ -47,6 +50,63 @@ func (ws *tbcWs) handlePingRequest(ctx context.Context, payload any, id string)
 	return nil
 }
 
+func (ws *tbcWs) handleBtcBlockMetadataByNumRequest(ctx context.Context, payload any, id string, db tbcd.Database) error {
```

Comment:
handle better for error cases here; add an `Error` field for user and internal errors, handle accordingly, and write tests